pymatgen/io/vasp/inputs.py

```diff
@@ -50,12 +50,6 @@
 logger = logging.getLogger(__name__)
 module_dir = os.path.dirname(os.path.abspath(__file__))
 
-# hashes computed from the full POTCAR file contents by pymatgen (not 1st-party VASP hashes)
-PYMATGEN_POTCAR_HASHES = loadfn(f"{module_dir}/vasp_potcar_pymatgen_hashes.json")
-# written to some newer POTCARs by VASP
-VASP_POTCAR_HASHES = loadfn(f"{module_dir}/vasp_potcar_file_hashes.json")
-POTCAR_STATS_PATH = os.path.join(module_dir, "potcar-summary-stats.json.bz2")
```

Comment:
I didn't really see a good reason to define these at such a "global" position when they are exclusively used by `PotcarSingle` and in the definition of `_gen_potcar_summary_stats` function. Meanwhile the `_gen_potcar_summary_stats` seems to be called only inside the unit test. Maybe `_gen_potcar_summary_stats` should be moved to the unit test as well? Is there any underlying thoughts I didn't know about @esoteric-ephemera? Thanks!

Reply:
I'm pretty sure that loading them here rather than every time a `PotcarSingle` is instantiated reduces load time. These were formerly loaded in at global spec, but that no longer is the case (not sure when that happened).

The `Potcar` class basically concatenates multiple `PotcarSingle` instances, so the way that these are loaded now might increase load time substantially. Probably smarter to load it in `Potcar` as well and feed that to `PotcarSingle`. Does that sound reasonable @Janosh?

`_gen_potcar_summary_stats` is/was used to generate the file potcar-summary-stats.json.bz2 and `PotcarSingle._potcar_summary_stats` attr. It could go into `dev_tools`, but we decided to keep it where it is for discoverability (#3370)

Reply:
Thanks a lot for the input @esoteric-ephemera 👍 . I'm not sure about the impact of above path definitions on the instantiation overhead of `Potcar` and `PotcarSingle`, but I would give it a test and discuss with you on the necessity of repositioning them then (in a separate PR because I think it might be better to make PRs more narrow on specific topic).